src/translators.js

```diff
@@ -1,24 +1,29 @@
 import curry from 'lodash.curry'
 
-export const tc = curry((interpolate, translate, message, scope = {}) => {
+import { interpolateComponents, interpolateString } from './interpolators.js'
+
+export const ti = curry((interpolate, translate, message, scope = {}) => {
   return interpolate(translate(message), scope)
 })
 
-export const tcn = curry((interpolate, translate, one, other, count, scope = {}) => {
+export const tin = curry((interpolate, translate, one, other, count, scope = {}) => {
   return interpolate(translate(one, other, count), scope)
 })
 
-export const tcp = curry((interpolate, translate, context, message, scope = {}) => {
+export const tip = curry((interpolate, translate, context, message, scope = {}) => {
   return interpolate(translate(context, message), scope)
 })
 
-export const tcnp = curry((interpolate, translate, context, one, other, count, scope = {}) => {
+export const tinp = curry((interpolate, translate, context, one, other, count, scope = {}) => {
   return interpolate(translate(context, one, other, count), scope)
 })
 
-const identity = x => x
+export const t = ti(interpolateString)
```

Comment:
We should probably add these to the documentation (`README.md`) and make it clear tc* is for components? Let me know if you want me to do a PR for this.

Reply:
We do need to update the README, yes. I'll give it a swing later!